src/agent/command_store/src/command_store.cpp

```diff
@@ -0,0 +1,93 @@
+#include <command_store.hpp>
+
+#include <iostream>
+
+namespace command_store
+{
+    CommandStore::CommandStore()
+        : m_tableName("Commands")
+        , m_dataBase(std::make_unique<sqlite_manager::SQLiteManager>("command_store.db"))
```

Comment:
The database name can be a const string.

Reply:
Done.